### ISSUE TITLE
[bug] cannot read property wordcount & blank existing projects when I updated to 0.5.1 fom 0.4.2

### ISSUE THREAD

**Issue by danieltomasz** [lynchjames/obsidian-writing-goals#16]
after updating to 0.5.1, i have problem with displaying the wring goals in the sidebar, I have blank pane and this error in the console
```
plugin:writing-goals:913 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'wordcount')
    at FrontmatterHelper.get (plugin:writing-goals:913:43)
    at NoteGoalHelper.getWordCount (plugin:writing-goals:964:52)
    at NoteGoalHelper.getWordCountRecursive (plugin:writing-goals:980:40)
    at NoteGoalHelper.getWordCountRecursive (plugin:writing-goals:986:44)
    at async NoteGoalHelper.createGoal (plugin:writing-goals:939:76)
    at async WritingGoals.loadNoteGoalData (plugin:writing-goals:8436:26)
    at async eval (plugin:writing-goals:8395:13)
g
```

**Comment by danieltomasz** [comment]
Pinning plug-in to 0.4.2 temporarily fixing this

**Comment by lynchjames** [comment]
Should be fixed in 0.5.2.
